charts/argocd-notifications/templates/_helpers.tpl

```diff
@@ -43,6 +43,18 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
 
+{{/*
+Common slack bot labels
+*/}}
+{{- define "argocd-notifications.bots.slack.labels" -}}
+helm.sh/chart: {{ include "argocd-notifications.chart" . }}
+{{ include "argocd-notifications.bots.slack.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
```

Comment:
Forgive me for asking but why duplicate `argocd-notifications.labels`?

Reply:
Because `argocd-notifications.bots.slack.labels` using `argocd-notifications.bots.slack.selectorLabels`.
They differs from argocd-controller label by `*-bot` suffix